src/main/resources/example-bags/valid/all-mappings/metadata/dataset.xml

```diff
@@ -32,6 +32,8 @@
                 <dcx-dai:titles>Dr</dcx-dai:titles>
                 <dcx-dai:initials>I</dcx-dai:initials>
                 <dcx-dai:surname>Lastname</dcx-dai:surname>
+                <dcx-dai:ISNI>0000 0001 2281 955X</dcx-dai:ISNI>
+                <dcx-dai:ORCID>https://orcid.org/0000-0001-9183-9538</dcx-dai:ORCID>
```

Comment:
In `easy-convert-bag-to-deposit` and `dd-ingest-flow` I see for both types of identifiers both numerical content as well as URLs. I wonder what would be the _preferred_ format.